documentation/slate/source/includes/interface.md

```diff
@@ -34,6 +34,10 @@ You can navigate through a response using 2 parameters
 |start_page|int |The page number          |
 |count     |int |Number of items per page |
 
+<aside class="notice">
+    Length (in characters) of the URL is limited to 4096 characters.
+</aside>
```

Comment:
will be printed in the `paging` section, why not in https://doc.navitia.io/#interface, just a bit upper in the page (before `paging`)?